syntax = "proto3";

package greet;

// The greeting service definition.
service Greeter {
    // Unary
    rpc SayHello (HelloRequest) returns (HelloReply);
}

// The request message containing the user's name and arrays.
message HelloRequest {
    string name = 1;
    string greeting = 2;
    repeated float weights = 3;
    repeated float biases = 4;
}

// The response message containing the greetings.
message HelloReply {
    string message = 1;
}
